entgql/annotation.go

```diff
@@ -211,6 +213,21 @@ func Type(name string) Annotation {
 	return Annotation{Type: name}
 }
 
+// TypeDescription returns a type description annotation.
+// TypeDescription annotation is used to add a descriptive comment to a GraphQL type.
+//
+// The below example adds a description to the 'ProcessPlan' type in the GraphQL schema.
+// Example:
+//
+//	func (ProcessPlan) Annotations() []schema.Annotation {
+//	   return []schema.Annotation{
+//	      entgql.TypeDescription("ProcessPlan represents the planning phase of a processing pipeline."),
```

Comment:
In our context it's rather "ProcessPlan represents a generic process corresponding to assembly or disassembly of a product." :)